src/features/movies/MovieList/styled.js

```diff
@@ -1,4 +1,4 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { Link } from "react-router-dom";
 
 export const Wrapper = styled.div`
```

Comment:
Czy to nie będzie footer naszej strony?

Reply:
Moim zdaniem mozna caly paginator zrobic jako footer, ale narazie warto dodac ta wersje a pozniej zajac sie rozmieszczeniem tego w dokumencie :)